web3j/src/main/java/com/vmware/ethereum/config/AppConfig.java

```diff
@@ -170,14 +173,21 @@ public TransactionManager transactionManager(
   }
 
   @Bean
-  public Web3j web3j(OkHttpClient okHttpClient, Credentials credentials) {
+  public Web3j web3j(OkHttpClient okHttpClient, Credentials credentials)
```

Comment:
Your IDE should tell you that `credentials` is unused variable and could be removed.